<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-EGGV59UT" failures="1" tests="4" name="Test2" time="11.068" errors="0" timestamp="2025-01-04T10:19:15 IST">
  <testcase classname="testng.ToverifytheLogin" name="navigate_to_web" time="3.666"/>
  <testcase classname="testng.ToverifytheLogin" name="usercredentials" time="0.865"/>
  <testcase classname="testng.ToverifytheLogin" name="click_on_login_button" time="1.325"/>
  <testcase classname="testng.ToverifytheLogin" name="logout" time="0.548">
    <failure type="java.lang.AssertionError" message="expected [https://opensource-demo.orangehrmlive.com/web/index.php/dashboard/index] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]">
      <![CDATA[java.lang.AssertionError: expected [https://opensource-demo.orangehrmlive.com/web/index.php/dashboard/index] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]
at testng.ToverifytheLogin.logout(ToverifytheLogin.java:83)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- logout -->
</testsuite> <!-- Test2 -->
